scripts/utils/price-utils.js

```diff
@@ -2,6 +2,10 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
   const axios = require("axios")
   const { fetchTokenInfoFromExchange } = require("./trading_strategy_helpers")(web3, artifacts)
 
+  const areBoundsReasonable = function(targetPrice, lowestLimit, highestLimit) {
+    return targetPrice * 0.5 < lowestLimit && highestLimit < targetPrice * 1.5
+  }
+
```

Comment:
Two comments:
1. the use is prompted to confirm because the price check failed but has no feedback on why. We should probably print a line explaining which check failed.
2 The range is not really symmetric, if you accept that the price goes down by a factor of two then arguably you accept that your price goes up by a factor of two.